pkg/collector/corechecks/servicediscovery/module/comm_test.go

```diff
@@ -132,31 +133,20 @@ func TestShouldIgnoreComm(t *testing.T) {
 
 			require.EventuallyWithT(t, func(collect *assert.CollectT) {
 				proc, err := customNewProcess(int32(cmd.Process.Pid))
-				require.NoError(t, err)
+				assert.NoError(t, err)
 
 				ignore := discovery.shouldIgnoreComm(proc)
-				require.Equal(t, test.ignore, ignore)
+				assert.Equal(t, test.ignore, ignore)
```

Comment:
suggestion, it can be moved outside the `require.Eventually` call, as it should not change over time

Reply:
I changed this unit test